includes/rest-api/class-sensei-rest-api-question-helpers-trait.php

```diff
@@ -335,6 +356,51 @@ private function get_question_common_properties( WP_Post $question ): array {
 			'editable'    => current_user_can( get_post_type_object( 'question' )->cap->edit_post, $question->ID ),
 			'categories'  => wp_get_post_terms( $question->ID, 'question-category', [ 'fields' => 'ids' ] ),
 		];
+
+		if ( ! empty( $question_meta['_question_media'][0] ) ) {
+			$question_media = $this->get_question_media( (int) $question_meta['_question_media'][0], $question->ID );
+
+			if ( ! empty( $question_media ) ) {
+				$common_properties['media'] = $question_media;
+			}
+		}
+
+		return $common_properties;
+	}
+
+	/**
+	 * Helper method to get question media.
+	 *
+	 * @param int $question_media_id The attachment id.
+	 * @param int $question_id       The question id.
+	 *
+	 * @return array Media info. It includes the type, id, url and title.
+	 */
+	private function get_question_media( int $question_media_id, int $question_id ) : array {
+		$question_media = [];
+		$mimetype       = get_post_mime_type( $question_media_id );
+		$attachment     = get_post( $question_media_id );
+
+		if ( $mimetype || null !== $attachment ) {
+			$mimetype_array = explode( '/', $mimetype );
+
+			if ( ! empty( $mimetype_array[0] ) ) {
+				if ( 'image' === $mimetype_array[0] ) {
+					// This filter is documented in class-sensei-question.php.
+					$image_size            = apply_filters( 'sensei_question_image_size', 'medium', $question_id );
+					$attachment_src        = wp_get_attachment_image_src( $question_media_id, $image_size );
+					$question_media['url'] = esc_url( $attachment_src[0] );
+				} else {
+					$question_media['url'] = esc_url( wp_get_attachment_url( $question_media_id ) );
+				}
+
+				$question_media['type']  = $mimetype_array[0];
+				$question_media['id']    = $attachment->ID;
+				$question_media['title'] = $attachment->post_title;
```

Comment:
Should this also be escaped?

Reply:
Escaped in 673fc50af0fb31a2cab1ec1784681ff83895e7bd